QuickReference.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0. Setup\n",
+    "Please make sure your environment is set up according to the instructions here: https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/00_SETUP.md\n",
+    "\n",
+    "Ensure you have the latest version of the workshop material by updating your environment:\n",
+    "TBD"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1. Overview\n",
+    "NASA services can be queried from Python in multiple ways.\n",
+    "* Generic Virtual Observatory (VO) queries.\n",
+    "  * Call sequence is consistent, including for non-NASA resources.\n",
+    "  * Use the `pyvo` package: https://pyvo.readthedocs.io/en/latest/\n",
+    "  * Known issues/caveats: https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/KNOWN_ISSUES.md\n",
+    "* Astroquery interfaces\n",
+    "  * Call sequences not quite as consistent, but follow similar patterns.\n",
+    "  * See https://astroquery.readthedocs.io/en/latest/\n",
+    "  * Informal Q&A session Tuesday, 5:30pm-6:30pm, NumFocus booth\n",
+    "* Ad hoc archive-specific interfaces\n",
+    "\n",
+    "# 2. VO Services\n",
+    "This workshop will introduce 4 types of VO queries:\n",
+    "* **VO Registry** - Discover what services are available worldwide\n",
+    "* **Simple Cone Search** - Search for catalog object within a specified cone region\n",
+    "* **Simple Image Access** - Search for image products within a spatial region\n",
+    "* **Simple Spectral Access** - Search for spectral products within a spatial region\n",
+    "* **Table Access** - SQL-like queries to databases\n",
+    "\n",
+    "## 2.1 Import Necessary Packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generic VO access routines\n",
+    "import pyvo as vo\n",
+    "\n",
+    "# For specifying coordinates and angles\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from astropy.coordinates import Angle\n",
+    "from astropy import units as u\n",
+    "\n",
+    "# For downloading files\n",
+    "from astropy.utils.data import download_file\n",
+    "\n",
+    "# Ignore unimportant warnings\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore', '.*Unknown element mirrorURL.*', vo.utils.xml.elements.UnknownElementWarning)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.1 Look Up Services in VO Registry\n",
+    "Simple example:  Find Simple Cone Search (conesearch) services related to SWIFT."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "services = vo.regsearch(servicetype='conesearch', keywords=['swift'])\n",
+    "services"
```

Comment:
Output of services is kind of ugly, and it doesn't seem like it serves a purpose here other than to show the query ran since you show how to iterate over the results later so maybe print services.status instead? (That would show "('OK', 'Query successful')")

Reply:
I like that!  Done.